src/config.rs

```diff
@@ -193,9 +193,11 @@ impl Config {
 
 fn clean_path_string(path_string: &str) -> path::PathBuf {
 	let separator_regex = regex::Regex::new(r"\\|/").unwrap();
+	let trailing_regex = regex::Regex::new(r"(/)+$").unwrap();
```

Comment:
Sure, I've brainfarted on these. But at the same time I believe Rust interpret these correctly. In fact, I think you wouldn't need any platform differences at all if dealing only with Path/PathBuf everywhere.